### ISSUE TITLE
stage3 traning

### ISSUE THREAD

**Issue by kuaileqipaoshui** [Chat-3D/Chat-3D-v2#5]
Thanks for your excellent work, but I have a question-why is stage3 pretrained_path using ckpt_00.pth instand of ckpt_02.pth？Are they the same?
![chat3d1](https://github.com/Chat-3D/Chat-3D-v2/assets/137666741/fe385646-d0ab-47d9-b053-bdbd1e7e28d7)

**Comment by ZzZZCHS** [comment]
We choose to use `ckpt_00.pth` based on the loss curve and validation metrics. Perhaps `ckpt_02.pth` would overfit more to the training data, but it depends. You can try it, and maybe you will get a better result. Good luck~

**Comment by kuaileqipaoshui** [comment]
> We choose to use `ckpt_00.pth` based on the loss curve and validation metrics. Perhaps `ckpt_02.pth` would overfit more to the training data, but it depends. You can try it, and maybe you will get a better result. Good luck~

thanks for your reply! I have some problems when I was in stage3.
like this:
![chat3d2](https://github.com/Chat-3D/Chat-3D-v2/assets/137666741/5e7a6627-3240-498c-92fa-11f53dee6cb0)
valueerror: could not convert string to float: b'3.0 8.0 1.0 4.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0'

Stage1 and Stage2 don't have any question, does that mean there is somthing wrong with the stage2_val400.json for the stage3 evaluation?Looking forward for your reply.

**Comment by kuaileqipaoshui** [comment]
> We choose to use `ckpt_00.pth` based on the loss curve and validation metrics. Perhaps `ckpt_02.pth` would overfit more to the training data, but it depends. You can try it, and maybe you will get a better result. Good luck~

It seems to be a problem with the evaluation, when I don't do a meteor evaluation, there is no problem, why? What is the purpose of the Phase 1 assessment (bleu, meteor, rouge, cider)? 
<img width="632" alt="chat3d3" src="https://github.com/Chat-3D/Chat-3D-v2/assets/137666741/f4de671e-8db4-441f-ad27-687116120580">
Is ckpt_00.pth selected based on the evaluation results?

**Comment by ZzZZCHS** [comment]
> > We choose to use `ckpt_00.pth` based on the loss curve and validation metrics. Perhaps `ckpt_02.pth` would overfit more to the training data, but it depends. You can try it, and maybe you will get a better result. Good luck~
> 
> thanks for your reply! I have some problems when I was in stage3. like this: ![chat3d2](https://private-user-images.githubusercontent.com/137666741/297707120-5e7a6627-3240-498c-92fa-11f53dee6cb0.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1ODg5NzcsIm5iZiI6MTcwNTU4ODY3NywicGF0aCI6Ii8xMzc2NjY3NDEvMjk3NzA3MTIwLTVlN2E2NjI3LTMyNDAtNDk4Yy05MmZhLTExZjUzZGVlNmNiMC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExOFQxNDM3NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYjI0NTVjNTU2NjFkNWNkNjVmMjkxMGE3ZDdiMjZkNTc4OTk1MmQ2OGY3ODYxM2E5NDg3NTY4NTlkMjQ0YTgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TTx2xOA7uYHz76p5Bp6yfHZDMtRCPvX263U_oxnzFIg) valueerror: could not convert string to float: b'3.0 8.0 1.0 4.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0'
> 
> Stage1 and Stage2 don't have any question.Looking forward for your reply.

This error can occur during the early steps of training when the model may generate some unusual and meaningless characters. You can refer to this [solution](https://github.com/salaniz/pytorch-gve-lrcn/issues/9#issuecomment-467083591). Or you can simply turn off the evaluation of the METEOR metric during the training process.

**Comment by kuaileqipaoshui** [comment]
> > > We choose to use `ckpt_00.pth` based on the loss curve and validation metrics. Perhaps `ckpt_02.pth` would overfit more to the training data, but it depends. You can try it, and maybe you will get a better result. Good luck~
> > 
> > 
> > thanks for your reply! I have some problems when I was in stage3. like this: ![chat3d2](https://private-user-images.githubusercontent.com/137666741/297707120-5e7a6627-3240-498c-92fa-11f53dee6cb0.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1ODg5NzcsIm5iZiI6MTcwNTU4ODY3NywicGF0aCI6Ii8xMzc2NjY3NDEvMjk3NzA3MTIwLTVlN2E2NjI3LTMyNDAtNDk4Yy05MmZhLTExZjUzZGVlNmNiMC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExOFQxNDM3NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYjI0NTVjNTU2NjFkNWNkNjVmMjkxMGE3ZDdiMjZkNTc4OTk1MmQ2OGY3ODYxM2E5NDg3NTY4NTlkMjQ0YTgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TTx2xOA7uYHz76p5Bp6yfHZDMtRCPvX263U_oxnzFIg) valueerror: could not convert string to float: b'3.0 8.0 1.0 4.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0'
> > Stage1 and Stage2 don't have any question.Looking forward for your reply.
> 
> This error can occur during the early steps of training when the model may generate some unusual and meaningless characters. You can refer to this [solution](https://github.com/salaniz/pytorch-gve-lrcn/issues/9#issuecomment-467083591). Or you can simply turn off the evaluation of the METEOR metric during the training process.

Thank you very much, but I still have a doubt. Just like the evaluation in 3Dqa and scene caption in your paper, should it be evaluated by checkpoint after the grounding fine-tuning or by checkpoint before the fine-tuning?
